RecoLocalMuon/GEMSegment/plugins/ME0SegFit.cc

```diff
@@ -0,0 +1,515 @@
+// ------------------------- //
+// --> ME0SegFit.cc 
+// Created:  11.05.2015
+// --> Based on CSCSegFit.cc
+// with last mod: 03.02.2015
+// as found in 750pre2 rel
```

Comment:
Copy-paste is not really so desirable.
I suppose that the developments may diverge.
[or/and ME0 become history]